### PR TITLE
Bug fix: Compilation error in PlayFabMultiplayerDataModels.m of the Objective-…

### PR DESCRIPTION
**Pull request by JTonoJimenez**
…C SDK

### CODE REVIEW

File: targets/objc/make.js [hunk 1]
```diff
@@ -282,10 +282,7 @@ function GetArrayPropertyDeserializer(property, datatype, api) {
     else if (property.isclass) {
         getter = "[[" + api.name +  property.actualtype + " new] initWithDictionary:[member_list objectAtIndex:i]]";
     }
-    else if (property.isenum) {
```
Comment (v-hoccha) [inline targets/objc/make.js]: I didn't fully investigated this, but I found some issues.
1 there are three more "else if (property.isenum)".
2. the enum type must call conversion  getter api(intValue or boolValue) to get real data from number object if it is required enum value.
3. the current implementation  is used this code to generate getter type, so I worry about the adding getting api and returning enum value directly can make some compile errors too if it is used to put the value is set to object.

**Comment by JTonoJimenez** [comment]
retest

**Comment by v-hoccha** [comment]
>         getter = "(" + api.name +  property.actualtype + ")" + "[member_list objectForKey:key]";

Can you check this code will be find?

---
Refers to: targets/objc/make.js:349 in 7b9b362. [](commit_id = 7b9b362bec11368af1422f8f4c60dd1066a65b05, deletion_comment = False)

**Comment by v-hoccha** [comment]
>         getter = "[[member_list objectForKey:key] boolValue]";

It seems like this can make compile error too.

---
Refers to: targets/objc/make.js:316 in 7b9b362. [](commit_id = 7b9b362bec11368af1422f8f4c60dd1066a65b05, deletion_comment = False)

**Comment by v-hoccha** [comment]
>         getter = "[[member_list objectAtIndex:i] boolValue]";

It seems like this can make compile error too.

---
Refers to: targets/objc/make.js:253 in 7b9b362. [](commit_id = 7b9b362bec11368af1422f8f4c60dd1066a65b05, deletion_comment = False)

**Comment by v-hoccha** [comment]
>         getter = "[[member_list objectForKey:key] boolValue]";

It seems like this can make compile error too.

---
Refers to: targets/objc/make.js:316 in 7b9b362. [](commit_id = 7b9b362bec11368af1422f8f4c60dd1066a65b05, deletion_comment = False)

**Comment by pgilmorepf** [comment]
Can you provide some kind of screenshot demonstrating that this project builds and/or executes some kind of useful output?

**Comment by v-hoccha** [comment]
>         getter = "(" + api.name + property.actualtype + ")" + "[properties valueForKey:@\"" + api.name + propName + "\"];";

This will also make compile error too.

---
Refers to: targets/objc/make.js:234 in 7b9b362. [](commit_id = 7b9b362bec11368af1422f8f4c60dd1066a65b05, deletion_comment = False)

**Comment by v-hoccha** [comment]
>         getter = "[[properties valueForKey:@\"" + propName + "\"] boolValue];";

It seems like this can make compile error too.

---
Refers to: targets/objc/make.js:201 in 7b9b362. [](commit_id = 7b9b362bec11368af1422f8f4c60dd1066a65b05, deletion_comment = False)

**Comment by pgilmorepf** [comment]
retest

**Comment by pgilmorepf** [comment]
retest

**Pull request merged**